Mathlib/RingTheory/Noetherian.lean

```diff
@@ -13,6 +13,7 @@ import Mathlib.Order.Filter.EventuallyConst
 import Mathlib.Order.OrderIsoNat
 import Mathlib.RingTheory.Finiteness
 import Mathlib.RingTheory.Nilpotent
+import Mathlib.RingTheory.GradedAlgebra.Basic
```

Comment:
I would guess that the issues in downstream files are caused by adding this import. I suggest you put the `IsNoetherianRing` result in a new `GradedAlgebra.Noetherian` file.